src/scripts/convert_lovd2.php

```diff
@@ -0,0 +1,1312 @@
+                    $aRecord[] = $sFieldValue;
+                } else {
+                    $aRecord[] = call_user_func($aFieldLinks[$sFieldName][2], $sFieldValue);
+                }
+            } else {
+                // Copy field as is.
+                $aRecord[] = $sFieldValue;
+            }
+        }
+
+        // Handle multiple observations for single variant in one patient
+        // (homozygous).
+        $sRecordID = $aInputRecord[array_search('ID_variantid_', $aInputHeaders)] . '_' .
+                     $aInputRecord[array_search('ID_patientid_', $aInputHeaders)];
+        if (array_key_exists($sRecordID, $aUniqueRecords)) {
+            // Combination variant_id/patient_id already seen, set previous
+            // record allele field to 3 (homozygous). Skip this record.
+            $aVOGRecords[$sRecordID]['allele'] = 3;
+            continue;
+        } else {
+            // Store current variant/patient combination for future reference.
+            $aUniqueRecords[$sRecordID] = null;
+        }
+
+        // Get submitter ID.
+        $sSubmitterID = null;
+        if (($i = array_search('ID_submitterid_', $aInputHeaders)) !== false) {
+            $sSubmitterID = $aRecord[$i];
+        }
+
+
+        // Create new disease if necessary.
+        $aDisease = lovd_getRecordForHeaders($aOutputHeaders['disease'], $aRecord,
+            $aSections['disease']);
+        if (!lovd_empty($aDisease['name'])) {
+            list($aDisease['id'], $bCreateNewDisease) = lovd_getDiseaseID($aDisease['name']);
+            if ($aDisease['id'] === false) {
+                // Something went wrong determining the disease, stop parsing.
+                break;
+            }
+            if ($bCreateNewDisease) {
+                // New disease, create an output record for it.
+                $aDiseases[] = $aDisease;
+            }
+        }
+
+        // Handle individual-specific data (individual, screening, phenotype, etc.).
+        $sIndividualInputHeader = 'ID_patientid_';
+        if (($i = array_search($sIndividualInputHeader, $aInputHeaders)) !== false) {
+            $sLOVD2IndividualID = $aInputRecord[$i];
+            if (!isset($aIndividuals[$sLOVD2IndividualID])) {
+                // New individual, create an output record for it.
+                $aIndividual = lovd_getRecordForHeaders($aOutputHeaders['individual'], $aRecord,
+                    $aSections['individual']);
+                if ($aIndividual['created_by'] === false) {
+                    // No curator ID was available, set submtiter ID.
+                    $aIndividual['created_by'] = lovd_convertUserID($sSubmitterID, 'submitter');
+                }
+                if ($aIndividual['edited_by'] === false) {
+                    // No curator ID was available, set submtiter ID.
+                    $aIndividual['edited_by'] = lovd_convertUserID($sSubmitterID, 'submitter');
+                }
```

Comment:
On second thought, `$aIndividual['edited_by'] === false` holds only when the input field was "0", set by `lovd_convertUserID()`. Empty edited_by fields will remain empty in the output.